scripts/deploy.sh

```diff
@@ -122,6 +127,10 @@ if [[ ! -f /etc/openstack_deploy/user_osa_secrets.yml ]] && [[ -f /etc/openstack
   mv /etc/openstack_deploy/user_secrets.yml /etc/openstack_deploy/user_osa_secrets.yml
 fi
 
+if [[ "$DEPLOY_MAGNUM" == "yes" ]]; then
+  cat $OA_DIR/playbooks/roles/os_magnum/extras/user_secrets_magnum.yml >> /etc/openstack_deploy/user_osa_secrets.yml
```

Comment:
The user_secrets_magnum.yml file can live in-tree here: https://github.com/rcbops/rpc-openstack/tree/mitaka-13.1/rpcd/etc/openstack_deploy
All yml files in this directory are copied over to /etc/openstack_deploy as part of some code run earlier in deploy.sh, so all you will have to do here is run ``pw-token-get.py`` similar to what we do here: https://github.com/rcbops/rpc-openstack/blob/master/scripts/deploy.sh#L162